java/util/pom_template.xml

```diff
@@ -32,7 +32,7 @@
             <Automatic-Module-Name>com.google.protobuf.util</Automatic-Module-Name> <!-- Java9+ Jigsaw module name -->
             <Bundle-DocURL>https://developers.google.com/protocol-buffers/</Bundle-DocURL>
             <Bundle-SymbolicName>com.google.protobuf.util</Bundle-SymbolicName>
-            <Export-Package>com.google.protobuf.util;version=${project.version}</Export-Package>
+            <Export-Package>com.google.protobuf.util;version={project.version}</Export-Package>
```

Comment:
since we see `com.google.protobuf.util;version=${project.version}` in the [generated pom](https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java-util/3.22.1/protobuf-java-util-3.22.1.pom) of `3.22.1`

i am thinking this needs to use `{version}` instead (see line 7)

if `{project.version}` would be available, the generated pom would have looked like this:
```
com.google.protobuf.util;version=$3.22.1
```

Reply:
i think both variants work, the plugin configuration probably does not even need to be in the published pom

what is important is that the version can be evaluated correctly when the build is generating the bundle, so that in the jar we see the right values in the `META-INF/MANIFEST.MF` file (snippet from `protobuf-java-util-3.21.12.jar`):
```
...
Bundle-SymbolicName: com.google.protobuf.util
Bundle-Version: 3.21.12
Created-By: Apache Maven Bundle Plugin
Export-Package: com.google.protobuf.util;version="3.21.12";uses:="com.go
 ogle.protobuf,javax.annotation"
...
```

this can be achieved by "hardcoding" the version during bazel replacement or by letting the bundle maven plugin evaluate `${project.version}` correctly (as done in the follow-up PR where `$` was added back)